src/search.ts

```diff
@@ -7,13 +7,16 @@ export const ioModules: string[] =
     ['http', 'fs', 'https', 'http2', 'net', 'datagram', 'child_process'];
 
 /**
- * requiredModules: A map of module names and the position where they were
- * required in a file dynamicEvals: An array of the locations where require is
- * called dynamically
+ * requiredModules: A map of I/O module names and the position where they were
```

Comment:
I think you should try to follow Google's TS style as best you can. Meaning parameters should be `@param paramName description`, and if you have to line wrap you should indent +4 spaces.

http://go/tsstyle#comments-documentation

Reply:
Looks like you do this later in the code, try to do so consistently :)

Reply:
I looked at the style guide and it doesn't say anything about how to the header for interfaces should look like. Are you sure I can use @param because ```requiredModules```, ```dynamicRequire```, ```dynamicArgs``` aren't necessarily parameters?

Reply:
Oh, haha thought this was a method due to the block of comments.

Generally for fields / methods on classes or interfaces you add the comment to each field. The interface / class can still have a comment if you like, but should talk about the whole thing, not every method.

```
/**
 * A pretty kitty
 */
interface Cat {
  /** Meows for your attention */
  meow();
}
```

You can leave out comments if they're self explanatory (so the above isn't the best example).